lib/externalGraderLocal.js

```diff
@@ -94,18 +99,24 @@ class Grader {
                     stderr: true,
                 }, (err, stream) => {
                     if (ERR(err, callback)) return;
-                    container.modem.demuxStream(stream, process.stdout, process.stderr);
+                    const out = byline(stream);
+                    out.on('data', (line) => {
+                        const newline = `container> ${line.toString('utf8')}`;
```

Comment:
What happens if utf-8 encoding fails? Does this throw an exception?

Reply:
The docs don't specify what happens, or even if it's possible to fail: https://nodejs.org/api/buffer.html#buffer_buf_tostring_encoding_start_end

Reply:
I tested it and it doesn't fail, it replaces invalid data with 0xfffd (https://unicode-table.com/en/FFFD/). So this is safe.